secant/Dependencies/DatabaseFiles/DatabaseFiles.swift

```diff
@@ -27,93 +27,57 @@ struct DatabaseFiles {
         self.fileManager = fileManager
     }
     
-    func documentsDirectory() throws -> URL {
+    func documentsDirectory() -> URL {
```

Comment:
I need to use `DatabaseFiles` when creating instance of `SDKSynchronizer`. And at that point it's not possible to handle exceptions. So I did this change. I think it should be ok.